apps/src/gamelab/GameLab.js

```diff
@@ -171,14 +183,18 @@ GameLab.prototype.init = function (config) {
     });
   }.bind(this);
 
-  ReactDOM.render(React.createElement(AppView, {
+  this.reduxStore_.dispatch(actions.setInitialLevelProps({
     assetUrl: this.studioApp_.assetUrl,
     isEmbedView: !!config.embed,
-    isShareView: !!config.share,
-    renderCodeWorkspace: renderCodeWorkspace,
-    renderVisualizationColumn: renderVisualizationColumn,
-    onMount: onMount
-  }), document.getElementById(config.containerId));
+    isShareView: !!config.share
+  }));
+
+  ReactDOM.render(<Provider store={this.reduxStore_}>
+    <GameLabView
+      renderCodeWorkspace={renderCodeWorkspace}
+      renderVisualizationColumn={renderVisualizationColumn}
+      onMount={onMount} />
```

Comment:
Done, in a fairly large renames-only change to stop saying "render" when it might be confused with React rendering.